accounts/views.py

```diff
@@ -202,6 +209,16 @@ def complete_profile(request):
 
     return render(request, 'accounts/volunteer_profile.html')
 
+def ajax_volunteer_rollcheck(request):
+    rollNo = request.POST['roll']
+    roll_no = Volunteer.objects.filter(roll_no=rollNo)
```

Comment:
You can name the first variable as `roll_no` and the second as `duplicate_rollchek`.